documentation/modules/overview/con-configuration-points-connect.adoc

```diff
@@ -5,127 +5,255 @@
 Kafka Connect instances are configured by default with the same:
 
 * Group ID for the Kafka Connect cluster
 * Kafka topic to store the connector offsets
 * Kafka topic to store connector and task status configurations
 * Kafka topic to store connector and task status updates
 
-If multiple different Kafka Connect instances are used, these settings must reflect each instance.
+If multiple different Kafka Connect instances are used, these settings must be unique for the workers of each Kafka Connect cluster created.
 
-[discrete]
-== Example YAML showing Kafka Connect configuration
+.Worker configuration that must be unique for each Kafka Connect instance
 [source,yaml,subs="attributes+"]
 ----
 apiVersion: {KafkaConnectApiVersion}
 kind: KafkaConnect
-metadata:
-  name: my-connect
+# ...
 spec:
-  # ...
   config:
     group.id: my-connect-cluster
     offset.storage.topic: my-connect-cluster-offsets
     config.storage.topic: my-connect-cluster-configs
     status.storage.topic: my-connect-cluster-status
-  # ...
 ----
 
-[discrete]
-== Connectors
+The group ID identifies the Kafka Connect worker nodes. Kafka Connect clusters cannot share the ID or topics as it will create errors.
+The names of the connectors used by each Kafka Connect instance must also be unique.
 
-Connectors are configured separately from Kafka Connect.
-The configuration describes the source input data and target output data to feed into and out of Kafka Connect.
-The external source data must reference specific topics that will store the messages.
+You can also deploy connector plugins using the `build` configuration of the `KafkaConnect` resource.
+Connector plugins provide specific configuration for the creation of connector instances.
 
-Kafka provides two built-in connectors:
+After connector plugins have been deployed, you can use Strimzi’s `KafkaConnector` custom resource or the Kafka Connect API to manage connector instances.
+You can also create new connector instances using these options.
 
---
-* `FileStreamSourceConnector` streams data from an external system to Kafka, reading lines from an input source and sending each line to a Kafka topic.
-* `FileStreamSinkConnector` streams data from Kafka to an external system, reading messages from a Kafka topic and creating a line for each in an output file.
---
+The Cluster Operator manages Kafka Connect clusters deployed using the `KafkaConnect` resource and connectors created using the `KafkaConnector` resource.
 
-You can add other connectors using connector plugins, which are a set of JAR files or TGZ archives that define the implementation required to connect to certain types of external system.
+[discrete]
+== Connector plugin configuration
+
+Connector plugins provide the configuration for connecting to an external data source.
+A set of JAR files or TGZ archives define the implementation required to connect to the data source.
```

Comment:
Since we support also ZIPs and Maven in the future, I would either use only JARs (since JARs are what is added at the end) or keep it completely generic.

Reply:
I made it generic